spec.md

```diff
@@ -0,0 +1,100 @@
+# bimg File Format Specification
```

Comment:
Potentially add a version and publication date here?

Reply:
First release is probably going to be 1.0.0, and the date will be appropriate when the standard is published.